lib/spack/docs/module_file_support.rst

```diff
@@ -367,14 +367,14 @@ and has similar effects on module file of dependees. Even in this case
 
   with the following snippet:
 
-  .. literalinclude:: ../../../var/spack/repos/builtin/packages/R/package.py
-     :pyobject: R.setup_environment
+  .. literalinclude:: ../../../var/spack/repos/builtin/packages/r/package.py
+     :pyobject: r.setup_environment
```

Comment:
This isn't going to work. The class is still named `R`, just like `python`'s class is named `Python`. You should remove this particular change.